Filipo0on/chessgame
# Feature/backend for await component

WHAT HAVE BEEN DONE

- The await page is now connected to the backend and reads data from there
- it uses the "GET_GAMES" query to get the information it needs (the list of all games currently in the database)
- the await page uses "PollInterval" to refetch the newest information from the backend (this is now set to refetch once every second)

HOW TO TEST

- checkout the branch
- run npm install in client folder if needed
- run npm install in the server if needed
-  start the json server("npm run json:server" in server folder)
- start the gql server("npm run dev" in server folder)
- start react on clientside ("npm start" in client folder)
- go to /await/idOfGameThatExists (example: /await/1)
-  go to the serverfolder->db.json,  and then change "gameStarted" to true on the game that you are on in the awaitscreen (example: if you are on /await/2 you should change on the game with id 2)
- confirm that the await page now routs to the game.
